client/src/modules/CrossCheck/ExportJSONButton.tsx

```diff
@@ -2,17 +2,29 @@ import { FileOutlined } from '@ant-design/icons';
 import { Button } from 'antd';
 import { CriteriaDto } from 'api';
 import _ from 'lodash';
+import { TaskType } from './components/CrossCheckCriteriaForm';
+import { CriteriaJSONType } from './UploadCriteriaJSON';
 
 interface Props {
   dataCriteria: CriteriaDto[];
 }
 
 export function ExportJSONButton({ dataCriteria }: Props) {
-  const removeKeyAndIndex = (criteria: CriteriaDto[]) => {
-    return criteria.map(item => _.omit(item, ['key', 'index']));
+  const transformCriteriaData = (criteria: CriteriaDto[]) => {
+    const transformedCriteria = criteria.map(item => {
+      let editedItem: Partial<CriteriaJSONType> = { ...item };
```

Comment:
Is it definitely necessary to use Partial here?

Reply:
Criteria from existing cross check lists has key 'title?' which is missing in CriteriaDto, and don't have keys 'key' and 'index' so I decided it would be ok to use here Partial

Reply:
ok